src/woo_publications/metadata/tests/test_organisation_api_endpoints.py

```diff
@@ -0,0 +1,480 @@
+            response = self.client.post(list_endpoint, headers={})
+
+            self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+        with self.subTest(action="update"):
+            response = self.client.put(detail_endpoint, headers={})
+
+            self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_list_organisations(self):
+        organisation = OrganisationFactory.create(
+            naam="one",
+            identifier="https://www.example.com/organisaties/1",
+            is_actief=True,
+            oorsprong=OrganisationOrigins.custom_entry,
+        )
+        organisation2 = OrganisationFactory.create(
+            naam="two",
+            identifier="https://www.example.com/organisaties/2",
+            is_actief=True,
+            oorsprong=OrganisationOrigins.municipality_list,
+        )
+        expected_first_item_data = {
+            "uuid": str(organisation.uuid),
+            "identifier": "https://www.example.com/organisaties/1",
+            "naam": "one",
+            "oorsprong": OrganisationOrigins.custom_entry,
+            "isActief": True,
+        }
+        expected_second_item_data = {
+            "uuid": str(organisation2.uuid),
+            "identifier": "https://www.example.com/organisaties/2",
+            "naam": "two",
+            "oorsprong": OrganisationOrigins.municipality_list,
+            "isActief": True,
+        }
+
+        response = self.client.get(
+            reverse("api:organisation-list"), headers=AUDIT_HEADERS
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        data = response.json()
+
+        self.assertEqual(data["count"], 2)
+        self.assertEqual(data["results"][0], expected_first_item_data)
+        self.assertEqual(data["results"][1], expected_second_item_data)
+
+    def test_list_organisations_filter_active(self):
+        organisation = OrganisationFactory.create(
+            naam="one",
+            identifier="https://www.example.com/organisaties/1",
+            is_actief=True,
+            oorsprong=OrganisationOrigins.custom_entry,
+        )
+        organisation2 = OrganisationFactory.create(
+            naam="two",
+            identifier="https://www.example.com/organisaties/2",
+            is_actief=False,
+            oorsprong=OrganisationOrigins.municipality_list,
+        )
+
+        expected_first_item_data = {
+            "uuid": str(organisation.uuid),
+            "identifier": "https://www.example.com/organisaties/1",
+            "naam": "one",
+            "oorsprong": OrganisationOrigins.custom_entry.value,
+            "isActief": True,
+        }
+        expected_second_item_data = {
+            "uuid": str(organisation2.uuid),
+            "identifier": "https://www.example.com/organisaties/2",
+            "naam": "two",
+            "oorsprong": OrganisationOrigins.municipality_list.value,
+            "isActief": False,
+        }
```

Comment:
we already test the serialization with the test above, so doing an assertion on `results[0]["identifier"]` is sufficient in these filter tests :)